app/models/setting/general.rb

```diff
@@ -22,7 +22,8 @@ def self.default_settings
       self.set('http_proxy_except_list', N_('Set hostnames to which requests are not to be proxied. Requests to the local host are excluded by default.'), [], N_('HTTP(S) proxy except hosts')),
       self.set('lab_features', N_("Whether or not to show a menu to access experimental lab features (requires reload of page)"), false, N_('Show Experimental Labs')),
       self.set("append_domain_name_for_hosts", N_("Foreman will append domain names when new hosts are provisioned"), true, N_("Append domain names to the host")),
-      self.set('outofsync_interval', N_("Duration in minutes after servers are classed as out of sync."), 30, N_('Out of sync interval'))
+      self.set('outofsync_interval', N_("Duration in minutes after servers are classed as out of sync."), 30, N_('Out of sync interval')),
+      self.set('widget_cache', N_("Cache for dasboard widgets in seconds, set to 0 to turn it off."), 30 * 60, N_('Widget cache'))
```

Comment:
is that 30 minutes cache? Imho that should be in the order of up to 5 minutes, no more, ideally a lot less imho